### ISSUE TITLE
MythPlugin's configure script uses deprecated python-distutuils

### ISSUE THREAD

**Issue by rcrdnalor** [MythTV/mythtv#437]
* **Platform**: any

* **MythTV version**: master

* **Package version**: n/a

* **Component**: MythTV Plugins

Excerpt from 'configure' script at
https://github.com/MythTV/mythtv/blob/991d0f47/mythplugins/configure#L465

If you run this function with python3 -W all,

check_py_lib()
...
from distutils.sysconfig import get_python_lib
...

it throws:
DeprecationWarning: The distutils package is deprecated and stated for removal in Python 3.12.
Use setuptools or check PEP 632 for potential alternatives.

This is the counterpart to #392 for the plugins:
"Python distutils is being removed".

**Comment by rcrdnalor** [comment]
Please look at PR #444:

This pull request fixes the build of MythNetVision for Debian/Ubuntu,
but does not enable the creation of a specific 'deb' package.
The 'configure' output shows now "MythNetvision  plugin will be built"
and it is built without errors.

It's future is still unclear, because MythNetVision depends on MythBrowser,
which is deprecated for QT6, see issue #327.

This commit will definitely work until python 3.12 is available.
See Python PEP 632 -- Deprecate distutils module

The need for a common python packaging concept and a proper build system
for python packages is definitely there for various OS-Distributions.
The OS Maintainers require that the python packages provided by
their distributions are clearly separated from the packages provided by
python pip.

Hopefully, the Python Package Authority (https://www.pypa.io)
and the OS distributions will better align in future.

Note: The MythTV Python Bindings is a type of less beloved packages,
because this package is tied to a specific version of MythTV and
not available on pip.

Reviews, suggestions and comments are welcome.

**Comment by rcrdnalor** [comment]
Starting with python 3.10+, Debian implements 
the following query to python-sysconfig correctly
See https://bugs.launchpad.net/ubuntu/+source/python3-defaults/+bug/1408092

```
$ python3
>>> import sysconfig

>>> sysconfig.get_scheme_names()
('deb_system', 'nt', 'nt_user', 'nt_venv', 'osx_framework_user', 'posix_home', 'posix_local', 'posix_prefix',
'posix_user', 'posix_venv', 'venv')

>>> sysconfig.get_paths()
{'stdlib': '/usr/lib/python3.11', 'platstdlib': '/usr/lib/python3.11', 
'purelib': '/usr/local/lib/python3.11/dist-packages', 
'platlib': '/usr/local/lib/python3.11/dist-packages', 'include': '/usr/include/python3.11', 
'platinclude': '/usr/include/python3.11', 'scripts': '/usr/local/bin', 'data': '/usr/local'}

>>> sysconfig._get_preferred_schemes()
{'prefix': 'posix_local', 'home': 'posix_home', 'user': 'posix_user'}

>>> sysconfig.get_preferred_scheme('prefix')
'posix_local'

>>> sysconfig.get_path('purelib')
'/usr/local/lib/python3.11/dist-packages'

>>> sysconfig.get_path('platlib')
'/usr/local/lib/python3.11/dist-packages'

>>> import sys
>>> sys.path
['', '/usr/lib/python311.zip', '/usr/lib/python3.11', '/usr/lib/python3.11/lib-dynload', 
'/usr/local/lib/python3.11/dist-packages', '/usr/lib/python3/dist-packages']
>>> 
>>> sysconfig.get_path('purelib', scheme='deb_system')
'/usr/lib/python3/dist-packages'

>>> sysconfig.get_path('platlib', scheme='deb_system')
'/usr/lib/python3/dist-packages'
>>> 

```

In other words, it is tine to switch to `sysconfig.get_path`.

**Comment by rcrdnalor** [comment]
Outdated PR #444 closed, new PR #776 opened because in the meantime,
the python package `sysconfig` is better maintained by the OS Distributions.
